soccer/main.py

```diff
@@ -14,16 +15,45 @@
 LEAGUE_IDS = leagueids.LEAGUE_IDS
 TEAM_NAMES = teamnames.team_names
 
+
+def write_config_file():
+    """Write API key to config.py file"""
+    print("No API key found!")
+    print("Please visit {0} and get an API token.".format(BASE_URL))
+    while True:
+        confkey = raw_input("Please enter API key: ")
+        if len(confkey) == 32:  # 32 chars
+            try:
+                int(confkey, 16)  # hexadecimal
+            except ValueError:
+                print("Invalid API key, try again")
+            else:
+                break
+        else:
+            print("Invalid API key, try again")
+    here = os.path.dirname(os.path.abspath(__file__))
+    with open(os.path.join(here, "config.py"), "wb") as conffile:
+        conffile.write('config = {\r\n')
+        conffile.write('    "SOCCER_CLI_API_TOKEN": "{0}",\r\n'.format(confkey))
+        conffile.write('}\r\n')
+    print("Config file written! Please restart Soccer CLI.")
+    sys.exit(2)
+
```

Comment:
Maybe something like this?
```
if sys.platform == "Win32":  # Windows
    subprocess.call("setx SOCCER_CLI_API_TOKEN {0}".format(confkey))
else:  # Mac/Linux:
    home = os.path.expanduser["~"]
    profile = os.path.join(home, ".profile")
    zprofile = profile.replace(".profile", ".zprofile")
    with open(profile, "a") as vfile:  # bash etc.
        vfile.write("export SOCCER_CLI_API_TOKEN={0}".format(confkey))
    if os.path.exists(zprofile):  # zsh
        with open(zprofile, "a") as vfile:
            vfile.write("export SOCCER_CLI_API_TOKEN={0}".format(confkey))
```
These are per-user. Global variables should instead use ``setx /M`` and ``/etc/profile.d/soccer-cli.sh``, unless we want to keep the API key on a per-user basis.